### PR TITLE
Navigation bar

### PR DESCRIPTION
**Pull request by FedericoChavazza**
## Links:

https://trello.com/c/6b8pUybq/11-users-can-navigate-the-site-using-the-navbar

## What & Why:

navbar integrated now can filter movie topics with mockup data

image

## Risks, Mitigation & Rollback:

- [x] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

**Review by martoesses — APPROVED** [review]

File: src/pages/Home/Homepage.js [hunk 1]
```diff
@@ -1,30 +1,56 @@
...
-        </div>
-        <div className="Homepage-container__Watchlist-title">
-          {' '}
-          <h1> {t('homepage.titles.watchlist')} </h1>
-          <h3> {t('homepage.titles.list')} </h3>
+      <div className="Homepage-container__content">
+        <div className="Homepage-container__carousel">
+          <div className="Homepage-container__button">
+            {' '}
+            <Button name="trending" handleClick={e => filterMovies(e)}>
```
Comment (JulianPasquale) [inline src/pages/Home/Homepage.js]: What about this? So we don't need the arrow function
```suggestion
            <Button name="trending" handleClick={filterMovies}>
```

Comment (FedericoChavazza) [reply src/pages/Home/Homepage.js]: Correcting that i realize that i had a bug where the state didn't changed as planned, turning into "undefined" not showing any carousel items for some reason, changing some things i realize that this is the better way to fix the problem:      
 <Button handleClick={() => filterMovies('trending')}> ===>  passing by parameter the actual value, it seems that the event.target.value sometimes comes with some delay

File: src/pages/Home/Homepage.js [hunk 2]
```diff
@@ -1,30 +1,57 @@
...
+  const [buttonValue, setButtonValue] = useState('discover');
+  const [buttonState, setButtonState] = useState({
+    trending: false,
+    discover: false,
+    top_rated: false,
+  });
+
+  const filterMovies = e => {
+    setButtonValue(e);
+    console.log(buttonValue);
```
Comment (JulianPasquale) [inline src/pages/Home/Homepage.js]: 👀

File: src/pages/Home/Homepage.js [hunk 3]
```diff
@@ -1,30 +1,57 @@
...
 import useTranslation from 'hooks/useTranslation';
+import Button from 'components/common/Button';
+import { useEffect, useState } from 'react';
 
 import { mockArrayData } from 'mock/mockData';
 
 export function HomePage() {
   const t = useTranslation();
+  const [buttonValue, setButtonValue] = useState('discover');
+  const [buttonState, setButtonState] = useState({
```
Comment (JulianPasquale) [inline src/pages/Home/Homepage.js]: Are you using this?

File: src/pages/Home/Homepage.js [hunk 4]
```diff
@@ -1,30 +1,57 @@
...
+    trending: false,
+    discover: false,
+    top_rated: false,
+  });
+
+  const filterMovies = e => {
```
Comment (JulianPasquale) [inline src/pages/Home/Homepage.js]: Do we need this function? Can we just call `setButtonValue` directly in the handler?
```javascript
<Button handleClick={() => setButtonValue('trending')}>
  ...
</Button>
```

File: src/pages/Home/Homepage.js (hunk 2 above)
Comment (FedericoChavazza) [reply src/pages/Home/Homepage.js]: asasdasdasd

File: src/pages/Home/Homepage.js (hunk 3 above)
Comment (FedericoChavazza) [reply src/pages/Home/Homepage.js]: noooo mb

File: src/pages/Home/Homepage.js (hunk 4 above)
Comment (FedericoChavazza) [reply src/pages/Home/Homepage.js]: actually idk why i did that function in the first place

**Review by JulianPasquale — APPROVED** [review]
Seems like you've created an empty file called [src/utils/api.js](https://github.com/FedericoChavazza/Movie-App/pull/5/files#diff-1952ef38ecd8505652ac151cac71f96e74b799a2e23344fa5bf2e74f3a813ae3).
Let's remove it from this PR

**Pull request merged**